docs/source/releases/v1_11_3a0.rst

```diff
@@ -10,14 +10,16 @@
  | # # # for more details. If you did not receive the license, for more information see:
  | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
 
-Version 1.11.3a0 (2023-08-19)
+Version 1.11.3a0 (2023-08-25)
```

Comment:
I am running full_test.sh now with these changes.  Once complete, we can approve and merge.  I will update the main summary with the test output once complete.

Reply:
Package: geoips_full
Total run time: 3130 seconds
Number data types run: 47
Number data types failed: 2

The only failures were from the missing ascat data - I am going to approve.